### PR TITLE
Game and Author Schema

### PR DESCRIPTION
**Pull request by OybekKayumov**
> In this pull request, I created:

 - the schema for game and author tables in `sql/schema.sql`.
 
issue: https://github.com/AyoubMs/catalog_of_my_things_ruby/issues/29

### CODE REVIEW

File: sql/schema.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+);
+
+-- create games table
+CREATE TABLE game (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  publish_date DATE,
+  archived BOOLEAN,
+  multiplayer BOOLEAN,  
+  last_played_at DATE,  
+  FOREIGN KEY (id) REFERENCES item (id) ON DELETE CASCADE
```
Comment (Ismailco) [inline sql/schema.sql]: When creating the tables I got this error `relation "item" does not exist` referencing this line `FOREIGN KEY (id) REFERENCES item (id) ON DELETE CASCADE  `

**Review by Ismailco — COMMENTED** [review]
#### Hi @OybekKayumov, 

Good job so far!
There are some issues that you still need to work on but you are almost there!
You did a great job, keep up the good work 👍🏼

Please check the inline comment below ⬇️ 

Cheers and Happy coding!👏👏👏

File: sql/schema.sql (hunk 1 above)
Comment (OybekKayumov) [reply sql/schema.sql]: create a table game updated,
thank you.
please see and give a feedback

**Review by Ismailco — APPROVED** [review]
### Hi @OybekKayumov, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

**Pull request merged**